.github/workflows/sync_develop_and_main.yml

```diff
@@ -0,0 +1,11 @@
+name: Create PR from main to develop
+
+on:
+  push:
+    branches: [main, master]
```

Comment:
since we don't have a `master` branch in this repo, we can just run on pushes to `main`

Reply:
Sure!  I was considering that.  I assume I should do that for headless as well.